### ISSUE TITLE
format date expression/effect

### ISSUE THREAD

**Issue by ismooch** [Njol/Skript#307]
If a variable is recognized as a time, it would be nice to effect how it shows. 

set {_time} to difference between now and {some.variable}
send {_time} to player as "HH Hrs MM Mins" 

or something like that. That may be a crazy request.. It just feels silly to execute that task everytime by parsing the date and forcing it to do what i want.

**Comment by Njol** [comment]
I.e. you want to be able to use a specific date or time format only at a certain location (or even only to be able to specify the format of a time span). Edited the title.

**Comment by ismooch** [comment]
@Njol yes.. thank you for understanding the idea. I use a lot of time variables from scoreboards to clocks to last vote times.. it is aggravating to have to parse and change the format forcibly.. not that I can not do it.. It would just be much easier, because adding multiple lines to turn "2 hours and 12 minutes" to "2 H & 12 M" just seems inefficient.
